src/rpc/CMakeLists.txt

```diff
@@ -7,5 +7,6 @@ find_package(fmt REQUIRED)
 find_package(gflags REQUIRED)
 
 add_library(mg-rpc STATIC ${rpc_src_files})
+add_library(mg::rpc ALIAS mg-rpc)
```

Comment:
I'm really happy to see that we started to use cmake `ALIAS`es.

Reply:
This is because in the past I've been burned by missing target and cmake just does `-ltargetname` can happened to find something called `targetname` that I didn't want to link against.

`ALIAS` ensure it must be a cmake target